test/integration/sharp/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "bun-integration-test-sharp",
+  "module": "index.ts",
+  "type": "module",
+  "scripts": {
+    "postinstall": "cd node_modules/sharp && bun install/check.js"
```

Comment:
this `postinstall` should be removed. if sharp doesnt work out of box then it is a bug in bun.